hooks/caret-position.js

```diff
@@ -0,0 +1,34 @@
+
+import {useState, useRef, useEffect, useCallback} from 'react'
+
+import useFocus from './focus'
+
+function useCaretPosition({initialValue, isFocus}) {
+  const [focusedElement] = useFocus({value: null, isFocus})
```

Comment:
Pourquoi ne pas passer `initialValue` ici ?
```suggestion
  const [focusedElement] = useFocus({value: initialValue, isFocus})
```